python/hail/docs/linalg/index.rst

```diff
@@ -1,6 +1,9 @@
 Linalg
 ======
 
+**File formats for numeric matrices are experimental**. Improvements
```

Comment:
I think the interface is experimental, too.

Reply:
Good point, updated.